multigeiger/multigeiger.ino

```diff
@@ -65,10 +66,19 @@ void setup() {
   setup_display(isLoraBoard);
   setup_switches(isLoraBoard);
   switches = read_switches();  // only read DIP switches once at boot time
+  if (!sendToBle)
+    switches.ble_on = false; // avoid switching on in WebConfig and trying to use BLE without setup
```

Comment:
that's a bit dirty, the switches.xxx should just reflect the hw switches.

Reply:
Hmm, any other idea how to prevent "using BLE" if not initialized? Another status variable will make the code uglier at many more calls.
My interpretation was, that we need indeed some "HW switch"...

Reply:
Resolved by BLE module's inner state variable